aurora-answers.rb

```diff
@@ -0,0 +1,58 @@
+# problem 1
+
+def mirror(list)
+
+  # returns a new array
+  # doubles size of orig array
+  # appends mirror image of original array on orig
+
+  result = []
+
+  list.length.times do |i|
+    result << list[i]
+  end
+
+  rev = -1
+  list.length.times do
+    result << list[rev]
+    rev -= 1
+  end
+
+  return result
+end
+
+# testing = [1,3,2,7]
+# puts mirror(testing)
+
+
+# problem 2
+
+
+def switchPairs(list)
+
+  result = []
```

Comment:
This solution was supposed to modify the array _in place_, not create a new copy of the array.